README.md

```diff
@@ -10,7 +10,7 @@ Visit [www.payara.fish](http://www.payara.fish) for full 24/7 support and lots o
 
 Information about the open source project is on the [GitHub project page](https://payara.github.io/Payara/).
 
```

Comment:
```suggestion
```

Reply:
Please remove this line completely. I know it's not related to Enterprise but this link no longer works and it's not relevant anymore.